qiskit/transpiler/passes/synthesis/plugin.py

```diff
@@ -336,6 +336,161 @@ def run(self, high_level_object, coupling_map=None, target=None, qubits=None, **
    HighLevelSynthesisPlugin
    HighLevelSynthesisPluginManager
    high_level_synthesis_plugin_names
+
+Available Plugins
+=================
+
+Below are the synthesis plugin classes available in Qiskit. These classes should not be
+used directly, but instead should be used through the plugin interface documented
+above. The classes are listed here to ease finding the documentation for each of the
+included plugins and to ease the comparison between different synthesis methods for
+a given object.
+
+
+Unitary Synthesis Plugins
+-------------------------
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   /qiskit/transpiler/passes/synthesis/unitary_synthesis/DefaultUnitarySynthesis
+   /qiskit/transpiler/passes/synthesis/solovay_kitaev_synthesis/SolovayKitaevSynthesis
```

Comment:
I think that `AQCSynthesisPlugin` also belongs here:
https://github.com/Qiskit/qiskit/blob/main/qiskit/transpiler/passes/synthesis/aqc_plugin.py

Reply:
should these docs replace the docs here: https://docs.quantum.ibm.com/api/qiskit/dev/transpiler_passes#synthesis
as it seems that there is some overlap between them